### DISCUSSION TITLE
Why is there a  difference between the simulation results on the RaidBots website and the simulation results on the SimulationCraft client?

### DISCUSSION THREAD

**Discussion by GonzoXing** [simulationcraft/simc#6183] [Q&A] [answered]
My preset condition is: 2 minutes of Pachwerk combat, all gains are checked. 
This is my character data
[致死连击.txt](https://github.com/simulationcraft/simc/files/7351903/default.txt)

**Comment by vituscze** [comment, marked as answer]
If you link to the Raidbots sim and upload simc_gui.simc (I'm assuming you're using the GUI client), then we can take a look at it. As it is, there's not enough information to give any meaningful answer.

**Reply by GonzoXing** [reply]
> If you link to the Raidbots sim and upload simc_gui.simc (I'm assuming you're using the GUI client), then we can take a look at it. As it is, there's not enough information to give any meaningful answer.

So when there is a difference between the two, which side is more credible?

**Reply by GonzoXing** [reply]
I suspect their Action lists are different, is there any chance of that ？

**Reply by vituscze** [reply]
Neither side is more credible, it's the same software. This is almost surely down to either: your local simc is out of date (Raidbots' simc is updated weekly/daily) or the sims used different settings (and yes, APL would be one of those).

**Reply by GonzoXing** [reply]
> Neither side is more credible, it's the same software. This is almost surely down to either: your local simc is out of date (Raidbots' simc is updated weekly/daily) or the sims used different settings (and yes, APL would be one of those).

I also think it is the difference of APL between the two.  I know how to look at client APL, but what about Raidbots APL?

**Reply by vituscze** [reply]
Run a sim and check the HTML report. There is a section which contains the APL.

**Reply by GonzoXing** [reply]
> Run a sim and check the HTML report. There is a section which contains the APL.

appreciate!
